src/settings/components/GeneralSettings.tsx

```diff
@@ -79,6 +79,13 @@ const GeneralSettings: React.FC<GeneralSettingsProps> = ({
         value={settings.defaultSaveFolder}
         onChange={(value) => updateSettings({ defaultSaveFolder: value })}
       />
+      <TextComponent
+        name="Default Conversation Tag"
+        description="The default tag to be used when saving a conversation. Default is 'ai-conversations'"
```

Comment:
I don't see where you set the default `ai-conversations` though, it's in constants.ts